pkg/profile/update.go

```diff
@@ -18,11 +19,24 @@ func Update(
 	argocdNs string,
 	arlonNs string,
 	profileName string,
-	bundlesPtr *string,
+	bundlesPtr []string,
 	desc string,
 	tags string,
 	overrides []arlonv1.Override,
 ) (dirty bool, err error) {
+	for _, name := range bundlesPtr {
+		if !bundle.IsValidK8sName(name) {
+			return false, fmt.Errorf("%w: %s", bundle.ErrInvalidName, name)
+		}
+	}
+	existingBundles, err := bundle.List(config, arlonNs)
+	if err != nil {
+		return false, err
```

Comment:
To build a cumulative string-based "stack trace" to help the caller identify the chain of calls leading to the error, don't just return the error unmodified, but instead return something like `fmt.Errorf("failed to list bundles: %s", err)`

Reply:
This is in profile/update, so could I add a message saying: `failed to get existing bundles` ?

Reply:
```
Error: failed to list bundles: failed to list secrets: Get "https://pf9-arlon.platform9.horse/qbert/v4/e5d5d7327829479ab04b050bf52a946e/sunpike/api/v1/namespaces/arlon/secrets?labelSelector=managed-by%3Darlon%2Carlon-type%3Dconfig-bundle": dial tcp: lookup pf9-arlon.platform9.horse: no such host
```

The error looks tike this now.